go-controller/pkg/config/config_test.go

```diff
@@ -513,6 +513,20 @@ service-cidr=172.18.0.0/24
 		Expect(err).NotTo(HaveOccurred())
 	})
 
+	It("invalid k8s-service-cidr results in an error", func() {
```

Comment:
(I don't know if the existing tests use this consistently, but note that `It()` actually prints "`It `" before the test description and so expects a description that completes a sentence, like `"throws an error if k8s-service-cidr is invalid"`. You can use `Specify()` instead of `It()` if you don't want that.)

Reply:
Fixed.